src/store/index.js

```diff
@@ -9,8 +9,7 @@ import datasets from '../datasets';
 import { isEqual, camelCase, uniqueId } from 'lodash-es';
 import { createFormatter } from './format';
 
-// eslint-disable-next-line import/no-webpack-loader-syntax
-import Worker from 'worker-loader!../worker';
+import Worker from '../layout.worker.js';
```

Comment:
This import cannot be resolved.
I'm assuming it should be
`import {layout as Worker} from '../worker';`

and export the `layout` function from the `src/worker/index.js`.

However if that is what should be imported, it takes 3 parameters when newing, but in this file it's instantiated as `new Worker()`, leading to runtime errors

Reply:
Oops, it looks like I forgot to properly track a moved file. Can you try again after updating to 
b37a958?

Reply:
It's working well now!
One thing I'd change is removing the `.js` extension from the import here, and in a couple of other places

Reply:
Good suggestion; see d5d46e7d6036469cc2d81c19c0411d1c614dd74a.